### PR TITLE
Custom validation

### PR DESCRIPTION
**Pull request by paul-porter**

### CODE REVIEW

File: dataduct/steps/executors/custom_check.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import pandas.io.sql as pdsql
```
Comment (recurlybrad) [inline dataduct/steps/executors/custom_check.py:1]: Would it be worth naming this file to `redshift_checks.py` or something similar indicating it contains redshift connection and query methods?

Or should this be considered the one place in dataduct where recurly custom checks will be defined?

File: dataduct/steps/executors/custom_check.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+def validate_redshift_result(sql, min_result, max_result):
+    result = get_redshift_data(sql)
+
+    if min_result and result < min_result:
+        raise Exception("result {0} is less than min {1}".format(result, min_result))
+    if max_result and result > max_result:
+        raise Exception("result {0} is greater than max {1}".format(result, max_result))
+
+    print result
```
Comment (recurlybrad) [inline dataduct/steps/executors/custom_check.py:21]: would it be useful for this method to return the result value at all?

**Review by recurlybrad — COMMENTED** [review]
Just a few comments about future usage.

File: dataduct/steps/executors/custom_check.py (hunk 1 above)
Comment (paul-porter) [reply dataduct/steps/executors/custom_check.py:1]: Yeah, that's probably a better name.

File: dataduct/steps/executors/custom_check.py (hunk 2 above)
Comment (paul-porter) [reply dataduct/steps/executors/custom_check.py:21]: I don't know how we would use a return value here. This function is called as a sort of callback from the custom step script in dataplatform. All I know how to do is throw an exception, or not throw one 😄

Comment (recurlybrad) [reply dataduct/steps/executors/custom_check.py:21]: oh yah, my comment doesn't make much sense now that I think about it.

**Review by mrcnc — APPROVED** [review]
👍

**Review by recurlybrad — APPROVED** [review]

**Pull request merged**